internal/server/auth/method/oidc/server.go

```diff
@@ -192,6 +194,7 @@ func (s *Server) providerFor(provider string, state string) (*capoidc.Provider,
 
 	p, err := capoidc.NewProvider(config)
 	if err != nil {
+		p.Done()
```

Comment:
this probably will panic because `p` is nil.

Reply:
Ah good call. Dunno what I was thinking